Source/SwiftLintFramework/Configuration.swift

```diff
@@ -112,7 +113,7 @@ public struct Configuration {
                 let yamlContents = try NSString(contentsOfFile: fullPath,
                     encoding: NSUTF8StringEncoding) as String
                 if let _ = Configuration(yaml: yamlContents) {
-                    print("Loading configuration from '\(path)'")
+                    fputs("Loading configuration from '\(path)'\n", stderr)
```

Comment:
I don't think custom reporters are particularly useful when format-breaking logs are printed to the same file descriptor. E.g. it's tricky to parse JSON from stdout if you first have to filter out the non-JSON parts.